bluesky/plans.py

```diff
@@ -3033,8 +2972,7 @@ def ramp_plan(go_plan,
 
         In seconds.
     '''
-    if md is None:
-        md = {}
+    md = {'plan_name': 'ramp_plan'}.update(md or {})
```

Comment:
That's reasonable. I chose the one liner because the two liner requires a new variable and more changes...

```python
_md = {'plan_name': 'ramp_plan'}
_md.update(md or {})
...
yield from blerg(md=_md)
```

but you convinced me it's worth it.